.idea/.gitignore

```diff
@@ -1,3 +0,0 @@
-# Default ignored files
```

Comment:
Seems worth adding `.idea/` to `.gitignore`

Off that's getting big fast.

Reply:
I assumed better to just remove the .idea/ (ie: pycharm files) entirely, rather than add specific code to .ignore? that was the approach I tried to take, although really they shouldn't have been committed at all. big goof on my part.